pkg/workloads/console/acceptance/acceptance.go

```diff
@@ -137,7 +174,7 @@ func (r *Runner) Run(logger kitlog.Logger, config *rest.Config) {
 			Eventually(func() error {
 				_, err = client.WorkloadsV1Alpha1().Consoles(namespace).Get(consoleName, metav1.GetOptions{})
 				return err
-			}, 12*time.Second).Should(HaveOccurred(), "expected not to find console, but did")
```

Comment:
Same reasoning as above.

Reply:
This case is a little different, as it's related to the 10-second `DefaultTTLSecondsAfterFinished` value, but I guess if we see it flake then we'll need to bump it too.

Reply:
Yeah let's add it back, I was probably a little heavy handed before removing this as well.